packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -1,13 +1,21 @@
 <template>
-	<div ref="sqlEditor" class="ph-no-capture"></div>
+	<div class="sql-editor">
```

Comment:
New components should ideally be done with Composition API?

Reply:
`sql-editor` class is unused?

Reply:
Will be addressed during editors refactoring that's in progress.